File: week2/c/ayush7.c
#include <stdio.h>
int main() {
    int p1[2]={3,2};
    int p2[2]={1,4};
    int res[3]={0};

    for(int i=0;i<2;i++)
        for(int j=0;j<2;j++)
            res[i+j]+=p1[i]*p2[j];

    printf("Product Polynomial: ");
    for(int i=0;i<3;i++)
        printf("%d ",res[i]);
    return 0;
}

File: week2/py/ayush7.py
p1 = [3, 2]  
p2 = [1, 4] 

res = [0]*(len(p1)+len(p2)-1)

for i in range(len(p1)):
    for j in range(len(p2)):
        res[i+j] += p1[i]*p2[j]

print("Product Polynomial:", res)
